test/e2e/framework/deployment.go

```diff
@@ -40,10 +40,10 @@ const SlowEchoService = "slow-echo"
 const HTTPBinService = "httpbin"
 
 type deploymentOptions struct {
-	namespace string
-	name      string
-	replicas  int
-	image     string
```

Comment:
why is this image being removed? Honestly asking, isn't this used anymore?

Reply:
not removed, look at whole diff, just added spaces to match longest struct field name

Reply:
ahh, image, yes it looks like not used variable anymore according my IDE.

Reply:
It looks like that var was  never used.